lib/obp/access/elements/paragraph.rb

```diff
@@ -14,7 +14,7 @@ def match_node?
 
           def content
             Nokogiri::XML::Builder.new do |xml|
-              xml.p { xml << sanitize_text(node.inner_html) } # Force xml tags generation rather than html escaping
+              xml.p sanitize_text(node.inner_html)
```

Comment:
Issue in introduction section, where some elements were rendered as xml where it was not supposed to be:
```
— <hydraulic> indicates that the term relates only to hydraulic technology;
— <pneumatic> indicates that the term relates only to pneumatic technology;
```

Reply:
@couraudt these `<...>` are called "domain", are you able to add the "domain" in the STS tree?

Reply:
Yes it is :) Here's a screenshot example of the output file:

![Screenshot From 2024-12-20 10-08-32](https://github.com/user-attachments/assets/ed436481-ca01-400b-98be-4f4b906e81ee)